ci/deploy/website.sh

```diff
@@ -33,7 +33,7 @@ declare -A shortlinks=(
 )
 
 cd doc/user
-hugo --gc --baseURL /docs --destination public/docs
+hugo --gc --baseURL /docs --destination public/docs/next
```

Comment:
```suggestion
hugo --gc --baseURL /docs --destination public/docs/unstable
```

Reply:
Guess it doesn't matter, in the end, but I do prefer `/unstable`.

Reply:
Fixed now.